js/actions/windowActions.js

```diff
@@ -729,6 +729,19 @@ const windowActions = {
   },
 
   /**
+   * Dispatches a close call to all frames in tab page
+   * The provided frame will be closed.
+   *
+   * @param {Object} framesList - List of frames
+   * @param {Object} framePropsList - List of frame properties to consider
+   */
+  tabPageClosed: function (framesList, framePropsList) {
+    framePropsList.forEach((frameProps) => {
```

Comment:
this is ok for now but we'll want to change this later to something that dispatches to the store and does a single action.  This would be pretty slow as is sometimes with a full tab page set.